commit a038bf2f0a6933c47627e03b4b42774cd9611213
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date:   Fri Oct 17 18:12:56 2014 -0400

    add lock-obj for new arch triplet for x86
    
    * src/sysconfig/lock-obj-pub.i586-pc-linux-gnu.h: New.
    
    --
    
    Helmut Grohne writes:
    
    Observe that the detected GNU triplet is now i586-pc-linux-gnu (gcc
    bumped it from i486-pc-linux-gnu recently), so the corresponding lock
    obj header in the syscfg folder needs to be moved or linked.
    
    Debian-Bug-Id: 764881

diff --git a/src/syscfg/lock-obj-pub.i586-pc-linux-gnu.h b/src/syscfg/lock-obj-pub.i586-pc-linux-gnu.h
new file mode 100644
index 0000000..fc2d132
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.i586-pc-linux-gnu.h
@@ -0,0 +1,23 @@
+## lock-obj-pub.i586-pc-linux-gnu.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[24];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -52,6 +52,7 @@
         syscfg/lock-obj-pub.i486-pc-gnu.h                   \
         syscfg/lock-obj-pub.i486-pc-kfreebsd-gnu.h          \
         syscfg/lock-obj-pub.i486-pc-linux-gnu.h             \
+        syscfg/lock-obj-pub.i586-pc-linux-gnu.h             \
         syscfg/lock-obj-pub.m68k-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.mips-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.mipsel-unknown-linux-gnu.h      \
